AQA_Lab_Task/CurrencyConvertor/ConsoleReader.cs

```diff
@@ -0,0 +1,51 @@
+namespace CurrencyConvertor;
+
+public static class ConsoleReader
+{
+    private static string? _text;
+    private static string? _text2;
+    private static string? _amount;
+    public static decimal Amount;
+    public static string? Currency;
+    public static string? СonversionСurrency;
```

Comment:
Fields for class - private. For your case - make fields private and create getters.